modules/common/react/lib/Popper.tsx

```diff
@@ -13,95 +13,95 @@ export interface PopperProps extends React.HTMLAttributes<HTMLDivElement> {
   /**
    * The children of the Popper (to be used as its content).
    */
-  children: React.ReactNode;
+  children: React.ReactNode | ((props: {placement: Placement}) => React.ReactNode);
   /**
    * The element that contains the portal children when `portal` is true.
    */
-  containerElement?: Element;
+  containerElement?: Element | null;
   /**
    * If true, set the Popper to the open state.
    * @default true
    */
-  open: boolean;
+  open?: boolean;
   /**
    * The placement of the popper relative to the `anchorElement`. Accepts `auto`, `top`, `right`, `bottom`, or `left`. Each placement can also be modified using any of the following variations: `-start` or `-end`.
    * @default bottom
    */
-  placement: Placement;
+  placement?: Placement;
   /**
    * The additional options passed to the Popper's `popper.js` instance.
    */
-  popperOptions?: PopperOptions;
+  popperOptions?: Partial<PopperOptions>;
   /**
    * If true, attach the Popper to the `containerElement`. If false, render the Popper within the DOM hierarchy of its parent.
    * @default true
    */
-  portal: boolean;
+  portal?: boolean;
 }
 
-export class Popper extends React.PureComponent<PopperProps> {
-  private popper: PopperJS | null;
+export const Popper = React.forwardRef<HTMLDivElement, PopperProps>(
+  (
+    {
+      anchorElement,
+      children,
+      containerElement,
+      open = true,
```

Comment:
Nit: @mannycarrera4 has been putting all of the default props at the top in #525. We have a linting rule that requires this for function params, it just doesn't catch destructuring (maybe we should look into that?)